UnitsNet.Tests/CustomCode/RotationalSpeedTests.cs

```diff
@@ -42,6 +42,15 @@ public class RotationalSpeedTests : RotationalSpeedTestsBase
 
         protected override double RevolutionsPerSecondInOneRadianPerSecond => 1/(2*Math.PI);
 
+        protected override double DegreesPerSecondInOneRadianPerSecond => (180 / Math.PI) * 1;
+
+        protected override double MillidegreesPerSecondInOneRadianPerSecond => ((180 / Math.PI) * 1)* 1E3;
+
```

Comment:
I prefer tests having pre-calculated values here, as it's too easy to fall in the trap of just using the reverse conversion expression from the implementation. I recommend you use an external conversion calculator as a sanity check.